docs/docs/README.md

```diff
@@ -3,25 +3,127 @@ prev: false
+- Compose steps into workflows and trigger on HTTP requests, schedules or app events
+
+Pipedream also provides easy to use services to solve common serverless and integration challenges including state management, execution rate and concurrency controls, large file support (up to 5TB) and more! 
+
+Watch a demo (4 mins):
+
+-->
+
+## Is Pipedream for me?
+
+We make it easy to connect APIs with code-level control when you need it — and no code when you don't. If you and your team want to stop writing boilerplate code, struggling with authentication and managing infrastructure for API integrations, then Pipedream is for you. 
+
+Developers with a working knowledge of Node.js or Javascript will get the most value from Pipedream (Python, Typescript. and GitHub integration are coming soon).
+
+<!--
+Pipedream is trusted by 150k+ developers from startups to Fortune 500 companies:
+
+![logos](https://res.cloudinary.com/pipedreamin/image/upload/v1612919944/homepage/logos_kcbviz.png)
+-->
+
+## Getting Started
+
+Sign up for a [free account (no credit card required)](https://pipedream.com/auth/signup) and complete our [quickstart guide](/quickstart/) to learn the basic patterns for workflow development:
+
+- Trigger workflows on HTTP requests, schedules and app events
+- Return a custom response from your workflow on HTTP requests 
+- Use connected accounts in actions and code steps
+- Pass data between code steps and no code actions
+- Use npm packages in Node.js code steps
+- Scaffold an API request in Node.js
+- End workflow execution early
+
+<!--
+<p style="text-align:center;">
+<a href="/quickstart/hello-world/"><img src="./quickstart/get-started.png"></a>
+</p>
+-->
+Pipedream offers a generous free tier (no credit card required) so you can test out the platform and use it for personal projects with no risk or committment. As your needs grow, it's easy to upgrade to paid plans to run without limits for individuals, teams (in alpha) and enterprises.
+
+## Use Cases
+
+Pipdream supports use cases from protototype to production and is trusted by 150k+ developers from startups to Fortune 500 companies:
+
+![logos](https://res.cloudinary.com/pipedreamin/image/upload/v1612919944/homepage/logos_kcbviz.png)
+
+The platform processes billions of events and is built and [priced](https://pipedream.com/pricing/) for use at scale. [Our team](https://pipedream.com/about) has built internet scale applications and managed data pipelines in excess of 10 million events per second (EPS) at startups and high-growth environments like BrightRoll, Yahoo!, Affirm and Instacart. 
+
+Our [community](https://pipedream.com/community) uses Pipedream for a wide variety of use cases including:
+
+- Connecting SaaS apps
+- General API orchestration and automation
+- Database automations (learn about [connecting to resources behind a firewall](/workflows/steps/code/nodejs/http-requests/#use-an-http-proxy-to-proxy-requests-through-another-host))
```

Comment:
We might not want to link here in this context, since that's HTTP-specific. I made a note to develop docs on what we typically suggest to users in this context: to run a bastion host and tunnel DB traffic through there.

Reply:
sounds good -- I changed it to a "reach out" message for now